canonical-web-and-design/ubuntu.com
# Removed the MAAS install links and updated the doc link on /download/server/provisioning

## Done

I removed the MAAS install steps and updated the docs link
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare [the copy doc](https://docs.google.com/document/d/12GXYcP9WrOgoso6Sf3bQql6nALRUzpdlcRW3Gbu-npM/edit#) with the [live page](https://ubuntu.com/download/server/provisioning) 


## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]
